style/tailwind.css

```diff
@@ -0,0 +1,416 @@
+  }
+  .footer-container__content {
+    @apply bg-primary text-gray-400 py-20 px-20 flex flex-col gap-9;
+  }
+  .footer-container__content-links {
+    @apply font-roboto grid grid-cols-5;
+  }
+  .footer-container__content-links-office {
+    @apply flex flex-col gap-3;
+  }
+  .footer-container__content-links-office-name {
+    @apply text-white font-bold paragraph-large;
+  }
+  .footer-container__link {
+    @apply cursor-pointer;
+  }
+  .footer-container__suscribe-news {
+    @apply flex flex-col gap-4 col-span-2 -mt-2;
+  }
+  .footer-container__logo {
+    @apply font-roboto-bold text-white text-3xl;
+  }
+  .footer-container__suscribe-news-content {
+    @apply w-full flex flex-col gap-4;
+  }
+  .footer-container__suscribe-news-title {
+    @apply text-white font-bold paragraph-large;
+  }
+  .footer-container__suscribe-news-input {
+    @apply flex gap-3 justify-between rounded-xsm bg-[#2B2E3C] w-full h-fit;
+  }
+  .footer-container__suscribe-news-email {
+    @apply outline-none px-4 font-roboto bg-transparent text-white;
+  }
+  .footer-container__designed-by {
+    @apply flex justify-center mt-5;
+  }
+  .footer-container__designed-by-line {
+    @apply w-full;
+  }
+  .footer-container__designed-by-name {
+    @apply w-full flex justify-around items-center;
+  }
+  .footer-container__designed-by-contact {
+    @apply flex gap-4;
+  }
+  .footer-container__designed-by-company {
+    @apply text-xs flex;
+  }
+  /* Footer Container end */
+  /* btn style start */
+  .btn,
+  .header-container__btn,
+  .conversion__btn,
+  .footer-container__trust-us-btn {
+    @apply font-roboto-bold font-extrabold text-white bg-primary hover:bg-light-violet hover:text-primary transition-all duration-500 ease-in-out cursor-pointer w-[144px] h-[56px] flex justify-center items-center rounded-xsm text-paragraph;
+  }
+
+  .btn--full {
+    @apply font-roboto-bold font-extrabold text-white bg-primary hover:bg-light-violet hover:text-primary transition-all duration-500 ease-in-out cursor-pointer w-full h-[56px] flex justify-center items-center rounded-xsm text-paragraph;
+  }
+
+  .btn--white,
+  .footer-container__suscribe-news-btn-white {
+    @apply font-roboto-bold font-extrabold text-primary bg-white hover:bg-primary hover:text-white transition-all duration-500 ease-in-out cursor-pointer w-[144px] h-[56px] flex justify-center items-center rounded-xsm text-paragraph;
+  }
+
+  .btn--outline,
+  .header-container__btn--outline {
+    @apply btn--white border-2 border-light-violet;
+  }
+  /* btn style  end */
+  .gradient-paragraph {
+    @apply text-gradient;
+  }
+
+  .paragraph-large,
+  .find-out-more__card-paragraph {
+    @apply text-[18px];
+  }
+}
+
+@font-face {
+  font-family: 'roboto';
+  src: url('/assets/Roboto-Regular.ttf');
+}
+
+@font-face {
+  font-family: 'roboto-bold';
+  src: url('/assets/Roboto-Bold.ttf');
+}
+
+.bg-gradient,
+.header-container__background--bg-gradient {
+  position: absolute;
+  width: 524px;
```

Comment:
should have used tailwindcss here instead of vanilla CSS in this class definition.